api/v1/annotations/cilium/annotations.go

```diff
@@ -1,6 +1,13 @@
 package cilium
 
 const (
+	// Make Cilium take ownership over the `/etc/cni/net.d` directory on the
+	// node, renaming all non-Cilium CNI configurations to `*.cilium_bak`.
+	// This ensures no Pods can be scheduled using other CNI plugins during Cilium
+	// agent downtime.
+	// Set this to "false" if you wish to use other CNIs such as Multus.
```

Comment:
let's remove this sentence - an annotation is typically just set or not set.

Reply:
The proposed patch actually expects a boolean value but we can drop that.